conf/application.conf

```diff
@@ -66,21 +66,12 @@ pega-uris {
     get-case = "/pega-proof-of-concept-stubs/case"
 }
 
-proxy {
-    username = pega-proof-of-concept-proxy
-    password = password
-    host = localhost
-    port = 9254
-    protocol = https
-    proxyRequiredForThisEnvironment = false
-}
-
 authDetails {
     username = "u"
     password = "1234"
 }
 
-http-verbs.proxy.enabled = true
+http-verbs.proxy.enabled = false
```

Comment:
In the http-verbs docs it says :
It uses WSProxyConfiguration.buildWsProxyServer which is enabled with http-verbs.proxy.enabled in configuration. It is disabled by default, which is appropriate for local development and tests, but will need enabling when deployed (if not already enabled by environmental configuration).
Would you not have to enable it in app-config-externaltest?

Reply:
it's set to `true` by the common config all services share in ET